egs/iban/s5/cmd.sh

```diff
@@ -0,0 +1,3 @@
+export train_cmd="run.pl --max-jobs-run 32"
+export decode_cmd="run.pl --max-jobs-run 32"
+export cuda_cmd="run.pl --max-jobs-run 4"
```

Comment:
g2.8xlarge
<https://www.google.com/search?client=safari&rls=en&q=g2.8xlarge&ie=UTF-8&oe=UTF-8>
does
(4)
the workshop expenses are calculated assuming that one.
y.

On Fri, Apr 22, 2016 at 5:52 PM, Vijayaditya Peddinti <
notifications@github.com> wrote:

> In egs/iban/s5/cmd.sh
> <https://github.com/kaldi-asr/kaldi/pull/727#discussion_r60808087>:
>
> > @@ -0,0 +1,3 @@
> > +export train_cmd="run.pl --max-jobs-run 32"
> > +export decode_cmd="run.pl --max-jobs-run 32"
> > +export cuda_cmd="run.pl --max-jobs-run 4"
>
> I don't think the AWS g2 instances have multiple GPUs per node.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/727/files/eedfdca7e9a3e78ba82aa49328bae01a46f09cd9#r60808087>
>